### PR TITLE
Modal column with comments

### PR DESCRIPTION
**Pull request by gizzmojr**
Split up the modal layout to be in columns. Following the styling queues of Github issue tracker, comments on the left and metadata on the right.

### CODE REVIEW

**Comment by gizzmojr** [comment]
Thinking of refactoring createSpinner, so I can add one to the comment section of the modal body. 

Seems hidden when in the background as is.
Thoughts?

**Comment by leif81** [comment]
Ya a spinner may help. But maybe there is also an underlying issue? The response times on our internal bz site seem slow. It's not apples and apples, but doing a comments query on a Mozilla bug is as much as 7x faster than instance. 300ms vs ~2000ms. At 300ms we could get away without a spinner. 😎 

https://bugzilla.mozilla.org/rest/bug/9998/comment
http://bugzilla.msec.local/rest/bug/14951/comment

Anything we can do server side to speed up our bz instance?

**Comment by gizzmojr** [comment]
Can you try non-FQDN for me, http://bugzilla/rest/bug/14951/comment
Mindful that Mozilla has proper web servers, load balancer, etc. Probably runs off the cloud too. 

Thinking the spinner still is a good idea while the page is doing work and something is displayed (modal).

Not a fan of having the comment box at the bottom thou.I I originally formatted it to be similar to the native BZ interface. Having to scroll down takes away from the speediness we're trying to maintain I feel.

**Comment by gizzmojr** [comment]
I like the hiding until ready, that feels more fitting and expected behavior

**Comment by leif81** [comment]
Ya changing it to bottom posting was a bit of an accident. But now that I'm conscious of it I see github and google inbox do that too :thinking:. It's more scrolling, but seeing what's being replied to for context is handy and seems more intuitive doesn't it? I'm tempted to change my bz user pref now!

**Comment by leif81** [comment]
@gizzmojr I tried the non-fqdn link, it's ~1s. Then I tried the original fqdn link and it's the same ballpark. Which is better than what I original was seeing, but still ~3x slower than the Mozilla link.

I think what we've got in this branch is good enough to merge, unless there's anything left you wanted to tackle?

**Comment by gizzmojr** [comment]
Ok now it looks good, found an issue with column widths for new bugs and when in resolved state.

**Review by leif81 — COMMENTED** [review]
Hmm this seems to break the case when a comment contains a really long line. The comment line previously would wrap. Not sure of the fix yet other than reverting. What was this one fixing?

**Comment by gizzmojr** [comment]
If the card action was anything other then click (show comments) the styling was all wrong. Meta width be 2/3 of the modal, description and/or add comment was tiny, etc. 
Your class (bug-comment) was doing all the work but wasn't for all cases. 

I see your issue. It's one continuous word is the issue, not a sentence. I'll investigate.

File: bzkanban.js [hunk 1]
```diff
@@ -1362,6 +1362,7 @@ function showBugModal(bugCurrent, bugUpdate) {
     meta.className = 'bug-meta';
 
     body.appendChild(comments);
+    showSpinner();
```
Comment (leif81) [inline bzkanban.js]: How come?

Comment (gizzmojr) [reply bzkanban.js]: Can be reverted, it was related to the discussion about slowness and I was adding some type of user notification. https://github.com/leif81/bzkanban/pull/57#issuecomment-276247231

In fact, could be switched out for a spinning cursor

Comment (leif81) [reply bzkanban.js]: Ah Ok, was just the position of it that threw me off. I think it'd be better to do it before the `httpGet` in that function bc I don't think we need a spinner for the "else" case (i.e. when a card is dragged).

Comment (gizzmojr) [reply bzkanban.js]: Thoughts on making it a cursor instead?

https://css-tricks.com/almanac/properties/c/cursor/ (progress or wait cursor)

Comment (leif81) [reply bzkanban.js]: Maybe ya. For now I added a commit to move the spinner logic in there.

**Comment by leif81** [comment]
I'm seeing one other case where the `bug-comment` div isn't right. If the content is less than our desired width it shrinks to that.

**Pull request merged**